### PR TITLE
implement RemoveShortcut

### PR DESCRIPTION
**Pull request by kevingentile**
### Description:

Addresses #1278 for shortcut cleanup

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

#### Where applicable:

- [x] Public APIs match existing style.
- [x] Any breaking changes have a deprecation path or have been discussed.

### CODE REVIEW

**Review by andydotxyz — DISMISSED** [review]

File: shortcut_test.go [hunk 1]
```diff
@@ -15,6 +15,22 @@ func TestShortcutHandler_AddShortcut(t *testing.T) {
 	assert.Equal(t, 2, len(handle.entry))
 }
 
+func TestShortcutHandler_RemoveShortut(t *testing.T) {
```
Comment (toaster) [inline shortcut_test.go:18]:
```suggestion
func TestShortcutHandler_RemoveShortcut(t *testing.T) {
```

File: shortcut_test.go [hunk 2]
```diff
@@ -15,6 +15,22 @@ func TestShortcutHandler_AddShortcut(t *testing.T) {
...
+
+	assert.Equal(t, 2, len(handle.entry))
+
+	handle.RemoveShortcut(&ShortcutCopy{})
+
+	assert.Equal(t, 1, len(handle.entry))
+
+	handle.RemoveShortcut(&ShortcutPaste{})
+
+	assert.Equal(t, 0, len(handle.entry))
```
Comment (toaster) [inline shortcut_test.go:31]:
```suggestion
	handler := &ShortcutHandler{}
	handler.AddShortcut(&ShortcutCopy{}, func(shortcut Shortcut) {})
	handler.AddShortcut(&ShortcutPaste{}, func(shortcut Shortcut) {})

	assert.Equal(t, 2, len(handler.entry))

	handler.RemoveShortcut(&ShortcutCopy{})

	assert.Equal(t, 1, len(handler.entry))

	handler.RemoveShortcut(&ShortcutPaste{})

	assert.Equal(t, 0, len(handler.entry))
```

**Review by toaster — CHANGES_REQUESTED** [review]
Looks good!
Just two minor typo/naming things, see suggestions.

File: shortcut_test.go (hunk 2 above)
Comment (kevingentile) [reply shortcut_test.go]: Note, this change is inconsistent with the existing tests in this file

**Comment by andydotxyz** [comment]
Looks good - @toaster can you please confirm if you are happy with the changes

**Comment by andydotxyz** [comment]
Looks llike the file was changed a lot so some merge conflicts to resolve @kevingentile sorry

**Comment by andydotxyz** [comment]
Static check failing:

/home/runner/work/fyne/fyne/shortcut.go
  (40, 2)  S1033  unnecessary guard around call to delete

**Review by andydotxyz — APPROVED** [review]

**Review by toaster — APPROVED** [review]

**Pull request merged**